tests/db.sh

```diff
@@ -340,29 +350,29 @@ main() {
 
 	# Exercise tests via GRPC
 	export TIGRIS_URL=localhost:8081
-	export TIGRIS_PROTOCOL=grpc
-	$cli config show | grep "protocol: grpc"
+	export TIGRIS_PROTOCOL=GRPC
```

Comment:
Why we need this change?

Reply:
this is not required change. It was left over during debugging and iterations. reverted case changes to small case.